net/aria2/files/aria2.init

```diff
@@ -341,6 +341,7 @@ aria2_start() {
 		procd_set_param user "$user"
 
 	procd_add_jail "$NAME.$section" log
+	procd_add_jail_mount "/usr/lib"
```

Comment:
This was why added now?

Reply:
see https://github.com/aria2/aria2/issues/2152#issuecomment-1848164124

Reply:
and I remove the comment and add now